pom.xml

```diff
@@ -187,6 +187,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.spotify</groupId>
+            <artifactId>hamcrest-optional</artifactId>
+            <version>1.1.5</version>
```

Comment:
Can we have the version number as a property?

Reply:
At a glance, it looks like except for things that share a version number, for singular entries like this we don't use version properties (e.g. mockito)